templates/optionalShibbolethLogin.tpl

```diff
@@ -0,0 +1,116 @@
+{**
```

Comment:
Another approach rather than replicating a modified template here would be to use a filter to modify the template as it is being rendered.  For example: orcidPlugin
https://github.com/pkp/orcidProfile/blob/e271e8801879bfce64e045f0110f1b8b8094b6bd/OrcidProfilePlugin.inc.php#L57-L58
https://github.com/pkp/orcidProfile/blob/e271e8801879bfce64e045f0110f1b8b8094b6bd/OrcidProfilePlugin.inc.php#L258-L262
https://github.com/pkp/orcidProfile/blob/e271e8801879bfce64e045f0110f1b8b8094b6bd/OrcidProfilePlugin.inc.php#L325-L354

This would allow multiple plugins to modify the login page, presenting additional login options.

Reply:
That's amazing. I didn't know that was a thing. I'll give that a shot.

Reply:
I really thought that the orcidProfile plugin already did this for the login page, but reading the code I only saw it on the registration page.  I might be missing it elsewhere.